src/views/wallet/Interfaces.ts

```diff
@@ -10,7 +10,7 @@ import { NetworkConfig } from './config'
 export interface Context {
     privkHex?: string
     privkCB58?: string
-    publicKey?: [Buffer, Buffer]
+    publicKey?: string
```

Comment:
Will it be a string ??

Reply:
Yes, I have changed it according to our needs